dataset/src/main/scala/frameless/ops/CubeOps.scala

```diff
@@ -0,0 +1,20 @@
+package frameless
+package ops
+
+import shapeless.ops.hlist.{ToTraversable, Tupler}
+import shapeless.{HList, HNil}
+
+class CubeManyOps[T, TK <: HList, K <: HList, KT](self: TypedDataset[T], groupedBy: TK)
```

Comment:
Maybe you could append these classes and the rollup ones to the `RelationalGroupsOps` file as they don't make much sense in isolation.